sormas-api/src/main/java/de/symeda/sormas/api/i18n/I18nProperties.java

```diff
@@ -338,14 +338,21 @@ private I18nProperties() {
 	}
 
 	public static String getContinentName(String defaultName) {
-		String name =
-			getInstance(userLanguage.get()).continentProperties.getString("continent." + defaultName.replace(" ", "_").toUpperCase() + ".name");
+		// remove spaces und everything in brackets following (e.g. "Australia (Continent)" => "AUSTRALIA")
+		defaultName = defaultName.substring(0, defaultName.contains("(") ? defaultName.indexOf("(") : defaultName.length())
+			.trim()
+			.replace(" ", "_")
+			.toUpperCase();
+		String name = getInstance(userLanguage.get()).continentProperties.getString("continent." + defaultName + ".name");
 		return name != null ? name : defaultName;
 	}
 
 	public static String getSubcontinentName(String defaultName) {
-		String name =
-			getInstance(userLanguage.get()).subcontinentProperties.getString("subcontinent." + defaultName.replace(" ", "_").toUpperCase() + ".name");
+		defaultName = defaultName.substring(0, defaultName.contains("(") ? defaultName.indexOf("(") : defaultName.length())
+			.trim()
+			.replace(" ", "_")
+			.toUpperCase();
```

Comment:
This cleanup and standardization is duplicate, please move to a single method with an appropriate name.

Reply:
This method is used for both continents and subcontinents and only does the cleanup to deliver part of the property name, Also,  follow proper CamelCasing.
==> `cleanContinentOrSubcontinentDefaultName` (?)